.github/CODEOWNERS

```diff
@@ -1 +1 @@
-* @Azure/azure-osconfig-core @MariusNi @robertschaedler3
+* @Azure/osconfig-maintainers
```

Comment:
>@Azure/osconfig-maintainers [](http://example.com/codeflow?start=2&length=27)

As discussed, add explicitly MariusNi after the team name.

Reply:
This will not change the notification behavior or PR requirements. Specifying just the team alias is the most clear and direct way to express our desired rules and will match will what we have internally on other projects.

Reply:
Regardless of notifications behavior and everything else stated, please put back MariusNi. And not just that, leave the azure-osconfig-team name as it was and add people to it.

Reply:
Closing this comment, we discuss this in at least another.